### PR TITLE
feat(analytics): add status color tokens [MA-2473]

### PR DESCRIPTION
**Pull request by mihai-peteu**
# Summary

https://konghq.atlassian.net/browse/MA-2473

3rd try's a charm 

See the Proposal 5 Final tab:

https://docs.google.com/spreadsheets/d/1RZyhXMJCr_bD5VNr3fubZx1isp3DmZGaL75TvYCPG-Q/edit?gid=80633302#gid=80633302

### CODE REVIEW

**Comment by mihai-peteu** [comment]
Awaiting Slack approval from UX, but otherwise ready to review.

File: tokens/source/components/status.json [hunk 1]
```diff
@@ -0,0 +1,341 @@
+{
+  "status": {
+    "color": {
+      "1na": {
+        "comment": "Analytics unknown 1xx request status color (blue.10).",
```
Comment (adamdehaven) [inline tokens/source/components/status.json:5]: If you all refer to it as `1xx` why is it named `1na`?

Comment (mihai-peteu) [reply tokens/source/components/status.json:5]: I'm just doing my best to appease both camps by using the `1xx` wildcard terminology in the comments, to appease those who didn't vote for `-na` ;)

Comment (adamdehaven) [reply tokens/source/components/status.json:5]: Tbh I think it makes it more confusing…

**Review by adamdehaven — COMMENTED** [review]
A couple general comments:

1. In the token descriptions, we shouldn't reference "Analytics" -- these status codes should be generic for "statuses" in general, so can we update the `comment` fields to read something like this:
    
    - (before): `Analytics unknown 2xx request status color (green.10).`
    - (after): `Color for an unknown 200-299 range response (green.10)`

2. For the "range tokens" (such as `400s`) can we update the comment to indicate it can be used for the whole range? Something like `Color to represent the response code range from 400-499 (orange.40)`
3. You will want to update [the stylelint plugin](https://github.com/Kong/design-tokens/blob/main/stylelint-plugin/rules/use-proper-token/token-map.mjs) to allow the use of the status tokens for a few properties. I'd recommend adding `'status-color'` to the following properties:

    - `background`
    - `background-color`
    - `color`
    - `fill`
    - `stroke`

File: tokens/source/components/status.json [hunk 2]
```diff
@@ -0,0 +1,299 @@
+{
+  "status": {
+    "color": {
+      "1na": {
+        "comment": "Color for an unknown 100-199 range response (blue.10).",
+        "value": "{color.alias.blue.10.value}"
+      },
+      "2na": {
+        "comment": "Color for an unknown 200-299 range response (green.10).",
```
Comment (adamdehaven) [inline tokens/source/components/status.json:9]: nit: change "response" to "response status code" in all the comments where appropriate

Comment (mihai-peteu) [reply tokens/source/components/status.json]: Updated to use "response status code" across the board

**Review by adamdehaven — APPROVED** [review]

**Pull request merged**